test/config/PayeConfigSpec.scala

```diff
@@ -22,14 +22,14 @@ import uk.gov.hmrc.play.test.UnitSpec
 
 class PayeConfigSpec extends UnitSpec with MockitoSugar with GuiceOneAppPerSuite {
   implicit val appConfig: ApplicationConfig = fakeApplication.injector.instanceOf[ApplicationConfig]
```

Comment:
use with Injecting and you will be able to remove this stuff to just inject[ApplicationConfig]